src/main/java/uk/gov/hmcts/sptribs/services/CaseManagementService.java

```diff
@@ -33,28 +34,28 @@ public CaseResponse createCase(String authorization, CaseData caseData) {
             }
 
             // creating case to CCD.
-            CaseDetails caseDetails = caseApiService.createCase(authorization, caseData,
+            final CaseDetails caseDetails = caseApiService.createCase(authorization, caseData,
                                                                 AppsUtil.getExactAppsDetails(appsConfig, caseData.getDssCaseData()));
             log.info("Created case details: " + caseDetails.toString());
             return CaseResponse.builder().caseData(caseDetails.getData())
                 .id(caseDetails.getId()).status(SUCCESS).build();
 
-
         } catch (Exception e) {
             log.error("Error while creating case." + e);
```

Comment:
Minor comment
Line number 44, 65, 66, 79 and 80 - a space can be added at the last.